src/components/BaseCheckbox/BaseCheckbox.tsx

```diff
@@ -0,0 +1,23 @@
+import computeClassNames from '../../utils/compute-class-names';
+import './BaseCheckbox.css';
+
+interface BaseCheckboxProps {
+  value: boolean;
+  onChange?: React.ChangeEventHandler<HTMLInputElement> | undefined;
```

Comment:
Mejorar la interfaz del a propiead onChange.
```suggestion
  onChange?(value: boolean): void;
```